drivers/d3d12/rendering_device_driver_d3d12.h

```diff
@@ -825,7 +825,7 @@ class RenderingDeviceDriverD3D12 : public RenderingDeviceDriver {
 	};
 
 public:
-	virtual RenderPassID render_pass_create(VectorView<Attachment> p_attachments, VectorView<Subpass> p_subpasses, VectorView<SubpassDependency> p_subpass_dependencies, uint32_t p_view_count) override final;
+	virtual RenderPassID render_pass_create(VectorView<Attachment> p_attachments, VectorView<Subpass> p_subpasses, VectorView<SubpassDependency> p_subpass_dependencies, uint32_t p_view_count, AttachmentReference p_fragment_density_map_attachment) override final;
```

Comment:
@DarioSamo , I think we can make an improvement here by always requiring the fragment density map attachment to be passed through like this. Yes it's a feature of the KHR FRS extension that we can set the density map on each subpass, but in practice for our use case, with out the mobile renderer using multiple subpasses, I think we can make the argument that we ALWAYS add the fragment density map to each subpass if FRS is used.

This would simplify the logic further on in the rendering engine and allow us to remove some of the enhancements around adding VRS attachments to subpasses. It will likely also make supporting Metal and DirectX easier as we won't need to make further checks whether those need to pass the attachment per subpass or as an extra parameter here.